cmd/descheduler/app/server.go

```diff
@@ -68,15 +69,21 @@ func NewDeschedulerCommand(out io.Writer) *cobra.Command {
 				klog.SetLogger(log)
 			}
 
+			ctx := context.TODO()
+			pathRecorderMux := mux.NewPathRecorderMux("descheduler")
 			if !s.DisableMetrics {
-				ctx := context.TODO()
-				pathRecorderMux := mux.NewPathRecorderMux("descheduler")
 				pathRecorderMux.Handle("/metrics", legacyregistry.HandlerWithReset())
+			}
+
+			// healthz returns true when health check pass
+			var checker []healthz.HealthChecker
 
-				if _, err := SecureServing.Serve(pathRecorderMux, 0, ctx.Done()); err != nil {
-					klog.Fatalf("failed to start secure server: %v", err)
-					return
-				}
+			checker = append(checker, healthz.NamedCheck("Descheduler", healthz.PingHealthz.Check))
+			healthz.InstallHandler(pathRecorderMux, checker...)
```

Comment:
You can just pass a single arg to variadic functions (no ned to create a slice)
```suggestion
			healthz.InstallHandler(pathRecorderMux, healthz.NamedCheck("Descheduler", healthz.PingHealthz.Check))
```

Reply:
+1

Reply:
Fix done, please check